### DISCUSSION TITLE
new Whatsapp API (configuration) encountered an error

### DISCUSSION THREAD

**Discussion by kylin-3** [Auties00/WhatsappWeb4j#98] [Q&A]
![image](https://user-images.githubusercontent.com/49175233/147814651-e7e0e98e-64df-4f98-b01f-366e2a61625d.png)
If these parameters are null, an exception will be thrown. How to solve it, thanks

**Comment by Auties00** [comment]
> ![image](https://user-images.githubusercontent.com/49175233/147814651-e7e0e98e-64df-4f98-b01f-366e2a61625d.png) If these parameters are null, an exception will be thrown. How to solve it, thanks

Those fields should not in fact be null. What are you trying to do?
